hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/dmf/amqp/DmfApiAutoConfiguration.java

```diff
@@ -27,13 +33,35 @@
 
     /**
      * Create default error handler bean.
-     * 
-     * @return the default error handler bean
+     *
+     *  @param handlers
+     *                  list of conditional error handlers
+
+     * @return the delegating error handler bean
      */
     @Bean
-    @ConditionalOnMissingBean
-    public ErrorHandler errorHandler() {
-        return new ConditionalRejectingErrorHandler();
+    @Primary
+    public ErrorHandler errorHandler(final List<ConditionalErrorHandler> handlers) {
+        return new DelegatingConditionalErrorHandler(handlers, new ConditionalRejectingErrorHandler());
     }
 
+    /**
+     * Error handler bean for all target related fatal errors
+     *
+     * @return the invalid target operations exception handler bean
+     */
+    @Bean
```

Comment:
Just as a hint, in case we need some handlers to run before others, we can add an `@Order` annotation to the beans
```java
    @Bean
    @Order(1)
    public ConditionalErrorHandler invalidTargetOperationsConditionalExceptionHandler() {
        return new InvalidTargetOperationsExceptionHandler();
    }

    @Bean
    @Order(2)
    public ConditionalErrorHandler messageConversionExceptionHandler() {
        return new MessageConversionExceptionHandler();
    }
```
But this is not needed here I think

Reply:
👍